test/smoketest/smoketest.bash

```diff
@@ -34,7 +34,7 @@ pkill -f java.-jar.target/colorado_rla || true
 dropdb corla || true
 createdb -O corla corla
 
-mkdir -p tarket
+mkdir -p target
```

Comment:
Thanks. I could have sworn I fixed that a while ago!
See also a fix for tabulations at https://github.com/FreeAndFair/ColoradoRLA/pull/510